spec/mailers/comment_mailer_spec.rb

```diff
@@ -2,6 +2,8 @@
   include MailerSpecHelper
   include EnvVarSpecHelper
 
+  let(:user) { create(:user) }
```

Comment:
yeah that's annoying -- if it were me I would put a `let` there and then inline `user` for the rest of the tests but I will merge as-is since I know this is related to failing jobs on prod.